Makefile

```diff
@@ -247,7 +247,7 @@ endif
 .PHONY: catalog-build
 ## Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
 catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool $(CONTAINER_TOOL) --mode replaces --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+	sudo $(OPM) index add --container-tool $(CONTAINER_TOOL) --mode replaces --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
```

Comment:
sudo?

Reply:
Merged as part of https://github.com/infinispan/infinispan-operator/pull/1382.

Unfortunately `sudo` is now required with their multi-arch images due to the way they have set the permissions. Previously we were using a single-arch tag, but that's no longer available so we have to work around this way.